src/requests/adapters.py

```diff
@@ -66,6 +68,20 @@ def SOCKSProxyManager(*args, **kwargs):
 DEFAULT_RETRIES = 0
 DEFAULT_POOL_TIMEOUT = None
 
+def C(name = None):
+    return name * 12
+
+C()
```

Comment:
**Suggestion:** The function C() is being called without any arguments, but it expects one. Also, it's not clear what the function C() is supposed to do. Consider renaming it to something more descriptive.
```suggestion
def multiply_by_twelve(name = None):
    return name * 12

multiply_by_twelve("example_argument")
```